coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
No, JSONEncoder, gets output of `__json__` which operates on class, so no values of properties here. Then it only filters out some values from a dict and that's it. The only thing that it gets is class attributes, nothing related to instances.